app/Views/components/layout.php

```diff
@@ -18,6 +18,8 @@
     <?php $this->renderSection('content') ?>
     
     <?php $this->renderSection('js') ?>
+    
+    <script src="/soucer/js/navbar.js"></script>
```

Comment:
```suggestion
    <script src="/assets/js/navbar.min.js"></script>
```
En terminos generales, todo lo que sea js, css e imagenes, siempre debe de apuntar hacia la ruta `/assets/` y solo colocas el nombre con el que fue generado el js o css, el cual siempre será `nombreArchivo.min.js` o `nombreArchivo.min.css`